extra/completions/_alacritty

```diff
@@ -18,7 +15,6 @@ _arguments \
   "--embed=[define the X11 window ID (as a decimal integer) to embed Alacritty within]:windowId" \
   "(-e --command)"{-e,--command}"[execute command (must be last arg)]:program: _command_names -e:*::program arguments: _normal" \
   "--config-file=[specify an alternative config file]:file:_files" \
-  "(-d --dimensions)"{-d,--dimensions}"[specify window dimensions]:columns: :lines" \
-  "--position[specify window position]:x position: :y position" \
+  "(-o --option)"{-o+,--option=}"[override config file options (example: cursor.style=Beam)]:option" \
```

Comment:
The options can also be specified multiple times, which you haven't taken into account. I'd propose just applying the following patch to the zsh completions:

```diff
diff --git a/extra/completions/_alacritty b/extra/completions/_alacritty
index 7f0d989..c97d563 100644
--- a/extra/completions/_alacritty
+++ b/extra/completions/_alacritty
@@ -15,6 +15,6 @@ _arguments \
   "--embed=[define the X11 window ID (as a decimal integer) to embed Alacritty within]:windowId" \
   "(-e --command)"{-e,--command}"[execute command (must be last arg)]:program: _command_names -e:*::program arguments: _normal" \
   "--config-file=[specify an alternative config file]:file:_files" \
-  "(-o --option)"{-o+,--option=}"[override config file options (example: cursor.style=Beam)]:option" \
-  "(-t --title)"{-t+,--title=}"[define the window title]:title" \
+  "*"{-o=,--option=}"[override config file options]:option" \
+  "(-t --title)"{-t=,--title=}"[define the window title]:title" \
   "--working-directory=[start shell in specified directory]:directory:_directories"
```